lib/node.js

```diff
@@ -86,8 +86,7 @@ Node.prototype.setInfo = function(data, callback)
 		{
 			this.trusted = true;
 		}
-
-		this.setGeo(data.ip);
+		this.setGeo(data.info.name);
```

Comment:
This line cannot be run because we never set `data.ip`. How about setting `ip` field instead in the `Node` function, and use `data.ip` instead?